MediaRecorder.bs

```diff
@@ -222,8 +222,8 @@ interface MediaRecorder : EventTarget {
       following steps:
       <ol>
         <li>Set {{state}} to {{inactive}}.</li>
-        <li><a>Fire an error event</a> named {{UnknownError}} at
-        <var>target</var>.</li>
+        <li><a>Fire an error event</a> named <a>error</a> with an
+        {{UnknownError}} at <var>target</var>.</li>
```

Comment:
I think we should fix the event order here instead.

It looks like these are the normative steps we already have for recording errors, here in step 5.4 under the [start](https://w3c.github.io/mediacapture-record/MediaRecorder.html#dom-mediarecorder-start) method: *"If the UA at any point is unable to continue gathering data for reasons other than isolation properties or stream track set, it MUST stop gathering data, and queue a task, using the DOM manipulation task source, that runs the following steps"*.